apollo-federation/src/query_plan/query_planner.rs

```diff
@@ -341,7 +348,7 @@ impl QueryPlanner {
             // belong to no subgraphs and use a special source named '_'. So we skip that "fake" source.
             let mut subgraphs = self
                 .federated_query_graph
-                .sources()
+                .source_schemas()
                 .filter(|&(name, _schema)| name != "_");
```

Comment:
This is already filtered out by `.source_schemas()` now.

Reply:
Good catch, removed the extra `filter` in the latest commit